corehq/apps/geospatial/utils.py

```diff
@@ -7,3 +14,46 @@ def get_geo_case_property(domain):
     except GeoConfig.DoesNotExist:
         config = GeoConfig()
     return config.case_location_property_name
+
+
+def get_geo_user_property(domain):
+    try:
+        config = GeoConfig.objects.get(domain=domain)
+    except GeoConfig.DoesNotExist:
+        config = GeoConfig()
+    return config.user_location_property_name
+
+
+def _format_coordinates(lat, lon):
+    return f"{lat} {lon} 0.0 0.0"
+
+
+def process_gps_values_for_case(domain, case_data):
```

Comment:
Nice! I have two thoughts:
1. I.M.O renaming this to something like `set_case_gps_property` / `set_gpt_property_value` and/or adding a small docstring will make it a tad bit clearer what the function does :smiley_cat: 
2. A rough idea: What if we move this function into the `CaseHelper` class? Or, we can create a new "GeoCaseHelper" class that is a subclass of `CaseHelper`, but exposes additional methods specifically to manage the geo property. Not 100% sure why I think this would be good, but the steps required to add/update a geo property here seems a bit convoluted. Let me know what you think :thinking:

Reply:
1. I agree this could be made a bit more clear, `set_case_gps_property` sounds good!
2. I wonder if we should also move the `process_gps_values_for_user` out somewhere? It feels a bit weird removing the case helper but keeping the user one. Perhaps we could move both of these out into a separate `PropertyHelper` class in the geospatial app. @SmittieC what do you think of doing it this way?

Reply:
Something like this @zandre-eng?

```python
class GeoPropertyHelper:
    def __init__(self, domain: str, lat: float, lon: float):
        self.domain = domain
        self.location_prop_name = get_geo_case_property(domain)
        self.coordinates = _format_coordinates(lat, lon)

    def set_user_coordinate(self, user_id):
        user = CommCareUser.get_by_user_id(user_id)
        metadata = user.metadata
        metadata[self.location_prop_name] = self.coordinates
        user.update_metadata(metadata)
        user.save()

    def set_case_coordinate(self, case_id):
        helper = CaseHelper(domain=self.domain, case_id=case_id)
        case_data = {
            'properties': {
                self.location_prop_name: self.coordinates
            }
        }

        helper.update(case_data)
```

I think the main benefit is the common operations being performed to 1. get the property name and 2. format that property value, given `lat` and `lon` values. Up to you if you want to do it this way. I'm fine either way :+1:

Reply:
Making a class to handle update of just one case property does look overkill here.
If there were more things being handle for geospatial feature on cases/users, making a class would have made sense.

Reply:
>I think the main benefit is the common operations being performed to 1. get the property name and 2. format that property value

@SmittieC We would still need to get the property name separately for users/cases since they could be different. Making it more DRY by centralizing the formatting value function would be nice, but Manish makes a good point that this might be unnecessary code overhead for what we're trying to achieve.

Given the above, I'm going to leave the functions in `utils.py` for now, and if we ever expand this functionality in the future then we can look at putting them into a helper class.